api/http/handler/endpointedge/endpointedge_status_inspect.go

```diff
@@ -155,6 +168,35 @@ func parseAgentPlatform(r *http.Request) (portainer.EndpointType, error) {
 	}
 }
 
+func (handler *Handler) handleSuccessfulUpdate(activeUpdateSchedule *updateschedule.EndpointUpdateScheduleRelation) {
+	handler.DataStore.EdgeUpdateSchedule().RemoveActiveSchedule(activeUpdateSchedule.EnvironmentID, activeUpdateSchedule.ScheduleID)
+	edgeStack, err := handler.DataStore.EdgeStack().EdgeStack(activeUpdateSchedule.EdgeStackID)
+	if err != nil {
+		log.Warn().
+			Err(err).
+			Msgf("Unable to find edge stack %d", activeUpdateSchedule.EdgeStackID)
```

Comment:
It's better to avoid using `Msgf()` and writing it like this instead:

```go
log.Warn().
	Int("edge_stack_id", activeUpdateSchedule.EdgeStackID).
	Msg("unable to find edge stack")
```